ultimatejobweb/admin.py

```diff
@@ -1,3 +1,5 @@
-# from django.contrib import admin
-
+from django.contrib import admin
+from .models import user, company
```

Comment:
You may want to automatically register all the models, so that you don't have to remember to register each one when you have a new one.
see: https://medium.com/hackernoon/automatically-register-all-models-in-django-admin-django-tips-481382cf75e5